admin/src/components/ChangeUserRoleFormular.spec.js

```diff
@@ -0,0 +1,257 @@
+import { mount } from '@vue/test-utils'
+import ChangeUserRoleFormular from './ChangeUserRoleFormular.vue'
+import { setUserRole } from '../graphql/setUserRole'
+import { toastSuccessSpy, toastErrorSpy } from '../../test/testSetup'
+
+const localVue = global.localVue
+
+const date = new Date()
```

Comment:
I think it is better to set `isAdmin: new Date()` every time you need it. So you do not have to look up what `date` is, when working on this test again.

Reply:
done